new-docs/content/any/getting-started/application/index.md

```diff
@@ -1,5 +1,11 @@
 ---
 title: Add Oso to an App (15 min)
 weight: 2
-any: false
+description: |
+  An in-depth walkthrough of adding Oso to an example expense application.
+aliases:
+  - /getting-started/application/index.html
 ---
+
+Add to your application coming soon. In the meantime, check out our
```

Comment:
I guess I don't know the difference besides this only requires one entry instead of a placeholder file in Go, Ruby and Rust.

Reply:
I would prefer the other way since then if someone doesn't have a language selected and goes to this page they'll see the language picker instead of `Add to your application coming soon.`, which might make them think that we don't have an example for the language they're interested in. If they're interested in Java, Python, or Node.js, that'd be an incorrect assumption!

Reply:
Ah, ok didn't realize that difference. Thanks will fix.

Reply:
Fixed here: https://github.com/osohq/oso/pull/769/commits/c248b5d8ecab7da994dba4a5d6528973e67744c9